llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -2827,12 +2830,59 @@ SDValue AMDGPUTargetLowering::lowerFEXPUnsafe(SDValue X, const SDLoc &SL,
                      Flags);
 }
 
+SDValue AMDGPUTargetLowering::lowerFEXP10Unsafe(SDValue X, const SDLoc &SL,
```

Comment:
What is "unsafe" lowering and how is it different? Is it faster but less accurate?
Can you add a comment to explain that?